docs/ref/files/storage.txt

```diff
@@ -95,6 +95,15 @@ The Storage Class
         able to return the last accessed time this will raise
         ``NotImplementedError`` instead.
 
+    .. method:: copy(src_name, dst_name)
+
+        Copy the file specified by ``src_name`` to ``dst_name``. If
+        ``dst_name`` already exists ``ValueError`` will be raised. For
+        storage systems that don't support copy this will raise
+        ``NotImplementedError`` instead.
+
+        .. versionadded:: 1.8
```

Comment:
master is now in feature freeze mode. See https://code.djangoproject.com/wiki/Version1.8Roadmap for the schedule and release process.